docs/dump.json

```diff
@@ -68,6 +68,10 @@
         "unit": "RPM",
         "value": 0
       },
+      "Fan Max": {
+        "unit": "RPM",
+        "value": 6000
```

Comment:
Saw you're defaulting it to 6k in here:
https://github.com/Umio-Yasuno/amdgpu_top/blob/ee770d7a6982647b1a9702cd2f30c384d3ce0eee/crates/amdgpu_top_gui/src/app.rs#L961

Reply:
Do not update `docs/*.json` in the PR as it will be regenerated later in my environment.

Reply:
I see, sorry, reverted it.